tests/test_backend.py

```diff
@@ -387,6 +378,13 @@ def expect_fail(test_case_path):  # type: (str) -> None
     expect_fail('OnnxBackendNodeModelTest.test_clip_default_max_cpu')
     expect_fail('OnnxBackendNodeModelTest.test_clip_default_min_cpu')
     expect_fail('OnnxBackendNodeModelTest.test_convtranspose_output_shape_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_1d_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_1d_exclusive_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_1d_reverse_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_1d_reverse_exclusive_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_2d_axis_0_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_2d_axis_1_cpu')
+    expect_fail('OnnxBackendNodeModelTest.test_cumsum_2d_negative_axis_cpu')
```

Comment:
It will be great if you could add comment why these tests are xfailing.

Reply:
```# Unsupported op: `CumSum```

Reply:
> @mitruska 
> `# Unsupported op: CumSum`

But CumSum is supported with static axis AFAIK.

Reply:
It's not visible on this fragment of diff. I removed expect_fail for CumSum tests in general and add it under [`if selected_backend_name == 'PlaidML':`](https://github.com/NervanaSystems/ngraph-onnx/blob/115f1d37feadc3d3d41f542dfddbcd828d83a1f2/tests/test_backend.py#L371) because there is no support for CumSum op in PlaidML that I concluded from plaidml [unit_test.manifest](https://github.com/NervanaSystems/ngraph/blob/e28110e6c120f371ab4bccf5df78fa2dd5ea256a/src/ngraph/runtime/plaidml/unit_test.manifest#L244-L252).